File: users-api/src/main/java/ar/com/miura/usersapi/controller/HealthController.java
package ar.com.miura.usersapi.controller;

import lombok.extern.slf4j.Slf4j;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@Slf4j
@RestController
public class HealthController {
    @GetMapping("/health")
    public String getStatus() {
        log.info("/health");
        return "ok";
    }
}

File: aws-sqs/src/main/kotlin/ar/com/miura/aws/sqs/controller/HealthController.kt
package ar.com.miura.aws.sqs.controller

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class HealthController {
    @GetMapping("/health")
    fun getStatus():String {
        return "ok";
    }
}